src/mono/wasm/README.md

```diff
@@ -50,25 +52,56 @@ The latest engines can be installed with jsvu (JavaScript engine Version Updater
 
 `npm install jsvu -g`
 
-* Run jsvu:
+* Run jsvu and install `v8`, `SpiderMonkey`,or `JavaScriptCore` engines:
 
 `jsvu`
 
 Add `~/.jsvu` to your `PATH`:
 
 `export PATH="${HOME}/.jsvu:${PATH}"`
 
+### Windows
+
+* Install node/npm from https://nodejs.org/en/ and add its npm and nodejs directories to the `PATH` environment variable
+
+* * Install jsvu with npm:
+
+`npm install jsvu -g`
+
+* Run jsvu and install `v8`, `SpiderMonkey`,or `JavaScriptCore` engines:
+
+`jsvu`
+
+* Add `~/.jsvu` to the `PATH` environment variable
+
 ## Libraries tests
 
 Library tests can be run with js engines: `v8`, `SpiderMonkey`,or `JavaScriptCore`:
 
+### Mac
+
 * `v8`: `make run-tests-v8-$(lib_name)`
 * SpiderMonkey: `make run-tests-sm-$(lib_name)`
 * JavaScriptCore: `make run-tests-jsc-$(lib_name)`
 * Or default: `make run-tests-$(lib_name)`. This runs the tests with `v8`.
 
 For example, for `System.Collections.Concurrent`: `make run-tests-v8-System.Collections.Concurrent`
 
+### Windows
+
+To run the libraries tests on windows, we have `RunTests` target. Without setting additional properties, it will run tests for all libraries on `V8` and `SpiderMonkey` engines:
```

Comment:
```suggestion
To run the libraries tests on windows, we have the `RunTests` target. Without setting additional properties, it will run tests for all libraries on `v8` and `SpiderMonkey` engines:
```